### PR TITLE
Add formula 5.6 from NEN-EN 1992-1-1+C2:2011

### LINKED ISSUE #155: [✨ Feature request]: Add formula 5.6 from NEN_EN_1992_1_1_C2_2011
### Description

Add formula 5.6 from NEN_EN_1992_1_1_C2_2011

Image

### Acceptance Criteria
- [ ] Add the formula to the code
- [ ] Add latex expression to the formula
- [ ] Add unittests
- [ ] Add docstrings
- [ ] Update table in docs with :heavy_check_mark:

### Dependencies

_not applicable_

### Confirmation

- [X] I have checked that this feature does not already exist in the issue tracker.
- [X] I have read and understood the contribution guidelines.

### PR DESCRIPTION
**Pull request by egarciamendez**
## Description

Add formula 5.6 from NEN-EN 1992-1-1+C2:2011

Fixes #155
afbeelding

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/test_formula_5_6.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        # https://docs.astral.sh/ruff/rules/try-except-in-loop/
+        # In this case, the desired behavior is to collect all the errors and raise them at once
+        for representation, values in latex_test_sample.items():
+            try:
+                assert values["expected"] == values["actual"]
+            except AssertionError as error_message:  # noqa: PERF203
+                assertion_errors.append(rf"Error in {representation} representation. {error_message}")
+
+        if assertion_errors:
+            raise AssertionError("{} errors occurred:\n{}".format(len(assertion_errors), "\n".join(assertion_errors)))
```
Comment (SZeltaat) [inline tests/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/test_formula_5_6.py:82]: To make things easier for you, I will add the suggestion here as well 😉 :

```suggestion
    @pytest.mark.parametrize(
        ("representation", "expected_result"),
        [
            ("complete", "H_{i} = Θ_{i} \cdot N_{a} = 0.003 \cdot 5.000 = 0.015"),
            ("short", "H_{i} = 0.015"),
            ("string", "H_{i} = Θ_{i} \cdot N_{a} = 0.003 \cdot 5.000 = 0.015"),
        ],
    )
    def test_latex(self, representation: str, expected_result: str) -> None:
        """Test the latex representation of the formula."""
        # Example values
        theta_i = 0.003  # -
        n_a = 5  # kN

        # Object to test
        form_5_6_latex = Form5Dot6TransverseForceEffectRoofDiaphragm(
            theta_i=theta_i,
            n_a=n_a,
        ).latex()

        actual = {
            "complete": form_5_6_latex.complete,
            "short": form_5_6_latex.short,
            "string": str(form_5_6_latex),
        }

        assert (
            actual[representation] == expected_result
        ), f"{representation} representation failed."
```

**Review by SZeltaat — COMMENTED** [review]
Hi Enrique,
I know you haven't asked me to review this PR. I just threw my suggestion to improve the test for latex representation. I haven't reviewed the rest of the PR.
Good luck!

**Review by bro-wi — APPROVED** [review]
LGTM

**Comment by egarciamendez** [comment]
> Hi Enrique, I know you haven't asked me to review this PR. I just threw my suggestion to improve the test for latex representation. I haven't reviewed the rest of the PR. Good luck!

I love your proactive approach. Thank you!

**Pull request merged**